cdk/cdk-resources/encryption-at-rest/src/index.ts

```diff
@@ -11,12 +11,14 @@ export interface CfnEncryptionAtRestProps {
   /**
    * @schema CfnEncryptionAtRestProps#AwsKms
    */
-  readonly awsKms: AwsKmsConfiguration;
```

Comment:
I guess this was manually renamed. 
cc @maastha

Reply:
Yes, "AwsKms" gives compilation error because of it being a keywork for JSII so we changed to"AwsKmsConfiguration". Does it compile successfully if you name it "AwsKms"? Can you try `npx projen compile` and share output?